docs/source/serving/openai_compatible_server.md

```diff
@@ -95,8 +95,7 @@ template, or the template in string form. Without a chat template, the server wi
 and all chat requests will error.
 
 ```bash
-python -m vllm.entrypoints.openai.api_server \
-  --model ... \
+vllm serve ... \
   --chat-template ./path-to-chat-template.jinja
 ```
 
```

Comment:
Based on #4709, the `:prog:` value under CLI args (line 111) should be updated to `vllm serve`.